runtimes/pytorch/ubi8-python-3.8/Pipfile

```diff
@@ -13,8 +13,8 @@ name = "pytorch"
 [packages]
 # PyTorch packages
 tensorboard = "~=2.14.0"
```

Comment:
just out of curiosity, why is here tensorboard 2.14 and in the other two it is 2.13?

Reply:
Usually this happen due to incopatibilities among the rest of the packages, i guess on that moment it was compatible with 
```
torch = {version = "~=2.0.0", index = "pytorch"}
torchvision = {version = "~=0.15.2", index = "pytorch"}
```
and python 8.